### PR TITLE
Wfat fixes

### PR DESCRIPTION
**Pull request by t-harps**

### CODE REVIEW

File: app/controllers/users/approvals_controller.rb [hunk 1]
```diff
@@ -22,4 +26,15 @@ def reject
     render "users/approvals/approve"
   end
 
+  def check_user
+    unless current_user?(params[:user_id])
+      flash[:notice] = "You are signed in as #{current_user.username}"
+      if params[:origin]
+        developer = Developer.find_by(company_name: params[:company_name])
```
Comment (t-harps) [inline app/controllers/users/approvals_controller.rb]: Couldn't think of a better way to get the right developer here, attempted to use X-API-KEY header but it's not present in this request and couldn't find a way to make it be.
Open to suggestions or just leave as is.

Comment (dansteele) [inline app/controllers/users/approvals_controller.rb]: @t-harps Perhaps make the API-KEY header mandatory for all requests? It potentially might make the api docs simpler

Comment (t-harps) [inline app/controllers/users/approvals_controller.rb]: This is what I tried to do but the request that sends the user to coposition is currently a redirect_to (so you can't set the headers). I tried replacing the redirect with an HTTParty get request but then WFAT complained about the controller not finding the correct route. It does feel like all requests should have the API-KEY header though so I might have another go.

Comment (dansteele) [inline app/controllers/users/approvals_controller.rb]: @t-harps That sounds more like a config problem: You should be able to send headers with any request :)

Comment (t-harps) [inline app/controllers/users/approvals_controller.rb]: @dansteele Couldn't find a way to add headers to the request issued by the redirect_to Coposition in the WFAT code. Couldn't find a way to replace the redirect_to with an HTTParty get request either so probably gunna stick with this.
@GarySiu merge this if it all looks ok.

**Pull request merged**